hive-lineage/test_hive_lineage.py

```
@@ -13,8 +15,15 @@ def __submit_hive_job(self, cluster_name):
   def verify_cluster(self, name):
     self.__submit_hive_job(name)
 
+  @parameterized.parameters(
+      'STANDARD',
+      'HA',
+  )
   def test_hive_job_success(self, configuration):
-    self.createCluster(configuration, self.INIT_ACTIONS, scopes='cloud-platform')
+    self.createCluster(configuration,
+                       self.INIT_ACTIONS,
+                       timeout_in_minutes=30,
```

Does it really need 30 mins? I think we should set with how long the test takes + 5 mins as buffer. Setting too much value here, doesn't allow us to fail fast.

---

Earlier the test was failing because of timeout, hence I had increased it to 30.

---

I removed it and tests still pass
(I think changing test size to "enormous" does the trick; otherwise it fails)